app/models/open_food_network/calculator/weight.rb

```diff
@@ -4,7 +4,7 @@ class Calculator::Weight < Spree::Calculator
     attr_accessible :preferred_per_kg
 
     def self.description
-      "Weight (per kg)"
+      I18n.t('calculator_weight_per_kg')
```

Comment:
If it works it would be exactly what I had experimented with doing. I think that the measurement system could be abstracted away from the name of the variable. I used `weight_measurement_unit` in my translation file. Here in the US we don't use the metric system (never heard a reasonable explanation back in the 3rd grade).